File: Maximum Count of Positive Integer and Negative Integer/Maximum Count of Positive Integer and Negative Integer.cpp
// Solution for Maximum Count of Positive Integer and Negative Integer in CPP

class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int pos = nums.end() - lower_bound(nums.begin(), nums.end(), 1);
        int neg = upper_bound(nums.begin(), nums.end(), -1) - nums.begin();
        return max(pos, neg);
    }
};

File: Maximum Count of Positive Integer and Negative Integer/Maximum Count of Positive Integer and Negative Integer.py
# Solution for Maximum Count of Positive Integer and Negative Integer in PY

class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        pos = len(nums) - bisect.bisect_left(nums, 1)
        neg = bisect.bisect_right(nums, -1)
        return max(pos, neg)
